src/Raven.Server/Documents/Sharding/Queries/Suggestions/ShardedSuggestionQueryProcessor.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Queries;
+using Raven.Client.Documents.Queries.Suggestions;
+using Raven.Client.Util;
+using Raven.Server.Documents.Queries;
+using Raven.Server.Documents.Queries.Sharding;
+using Raven.Server.Documents.Queries.Suggestions;
+using Raven.Server.Documents.Sharding.Commands.Querying;
+using Raven.Server.Documents.Sharding.Handlers;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Sharding.Queries.Suggestions;
+
+public class ShardedSuggestionQueryProcessor : AbstractShardedQueryProcessor<ShardedQueryCommand, QueryResult, SuggestionQueryResult>
+{
+    private readonly long? _existingResultEtag;
+
+    private readonly string _raftUniqueRequestId;
+    public ShardedSuggestionQueryProcessor(TransactionOperationContext context, ShardedDatabaseRequestHandler requestHandler, IndexQueryServerSide query, long? existingResultEtag, CancellationToken token) : base(context, requestHandler, query, false, false, token)
```

Comment:
can we used named parameters for bools? I do not know what `false, false` is

Reply:
Handled in https://github.com/ravendb/ravendb/pull/15718